python/cudf/cudf/tests/test_udf_masked_ops.py

```diff
@@ -682,6 +714,230 @@ def f(x):
     assert precompiled.currsize == 1
 
 
+@string_udf_test
+@pytest.mark.parametrize("data", [{"str_col": string_data}])
```

Comment:
As in the `string_udf` tests, I think judicious use of parametrized pytest fixtures would simplify this test suite.